projects/storefrontlib/src/shared/components/table/table.model.ts

```diff
@@ -18,27 +17,42 @@ export interface Table<T = any> {
    * to be unaware of any specifics. The type can be handed in when the `Table` is
    * created however.
    */
-  data$: Observable<T[]>;
+  data: T[];
```

Comment:
The JSDoc above says `The table data is provided in an observable, so that we can observe changes and benefit from change detection`. We should change the doc.